demos/llm_npu/README.md

```diff
@@ -38,8 +38,10 @@ Run `export_model.py` script to download and quantize the model:
 
 **LLM**
 ```console
-python export_model.py text_generation --source_model meta-llama/Llama-3.1-8B-Instruct --target_device NPU --config_file_path models/config.json --model_repository_path models  --overwrite_models
+python export_model.py text_generation --source_model meta-llama/Llama-3.1-8B-Instruct --target_device NPU --config_file_path models/config.json --ov_cache_dir ./models/.ov_cache --model_repository_path models  --overwrite_models
```

Comment:
Failed on import: 
E           RuntimeError: Exception from src/inference/src/cpp/core.cpp:95:
E           Exception from src/inference/src/model_reader.cpp:154:
E           Unable to read the model: models/meta-llama/Llama-3.1-8B-Instruct/openvino_model.xml Please check that model format: xml is supported and the model is correct. Available frontends: jax tf onnx pytorch paddle ir tflite